Brewfile

```diff
@@ -0,0 +1,5 @@
+brew "carthage"
+brew "sourcery"
+brew "swiftgen"
+brew "swiftlint"
```

Comment:
SwiftLint shouldn't be required anymore, since PR https://github.com/wireapp/wire-ios/pull/653

Reply:
Thanks!